Content.Server/StationEvents/Events/SolarFlare.cs

```diff
@@ -0,0 +1,86 @@
+using Content.Server.GameTicking.Rules.Configurations;
+using Content.Server.Radio.Components;
+using Content.Server.Radio;
+using Robust.Shared.Random;
+using Content.Server.Light.EntitySystems;
+using Content.Server.Light.Components;
+
+namespace Content.Server.StationEvents.Events
+{
+    public sealed class SolarFlare : StationEventSystem
+    {
+        [Dependency] private readonly PoweredLightSystem _poweredLight = default!;
+
+        public override string Prototype => "SolarFlare";
+
+        private bool _onlyJamHeadsets = true;
+        private HashSet<string> _affectedChannels = new();
+        private float _endAfter = 0.0f;
+        private bool _running = false;
```

Comment:
Okay im now pretty sure you can check RuleStarted when needed instead of storing this now